whatsnowplaying/whats-now-playing
# More tests and more bugfixes as a result of testing

* switch to calling pytest directly after fixing pyproject
* add a workaround for Mac OS X cfprefsd during testing
* add an autouse fixture to remove old test config file
* add no cover pragmas for now:
  * rare error conditions
  * systemtray
* fix a bad bug in bootstrap's upgrade code: open the
  qsettings file _directly_ to see what has and has
  not been set!
* add ability to specify a testdir for upgrade code
* add better logging all over the place
* remove Class level vars from Serato and the necessary
  rework for that
  * there is still a conflict between local and remote
    defaults in seratohandler and the plugin that should
    probably be clarified or resolved at some point
* add tests for serato remote
* add tests for m3u and fixed several bugs
* add tests for most of utils
